src/qibo/backends/numpy.py

```diff
@@ -75,6 +77,12 @@ def to_numpy(self, x):
     def compile(self, func):
         return func
 
+    def clifford_pre_execution_reshape(self, state):
+        return state
+
+    def clifford_post_execution_reshape(self, state, nqubits):
+        return state
```

Comment:
Why are these functions not doing anything?

Reply:
because for the `numpy` backend you don't need to do anything, this is used by the `cupy` backend only for reshaping the symplectic matrix

Reply:
I see. So they're overwritten in `qibojit`?

Reply:
yes they are